src/components/input-number/nz-input-number.component.ts

```diff
@@ -36,16 +41,18 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
     <div
       class="ant-input-number-input-wrap">
       <input class="ant-input-number-input"
-        #inputNumber
-        (blur)="onTouched();_checkValue()"
-        [placeholder]="nzPlaceHolder"
-        [disabled]="nzDisabled"
-        [(ngModel)]="_displayValue"
-        (ngModelChange)="_userInputChange()"
-        [attr.min]="nzMin"
-        [attr.max]="nzMax"
-        [attr.step]="_step"
-        autocomplete="off">
+             #inputNumber
```

Comment:
Beside the code you add, why did you shift the code?
It seems no need to do this.

Reply:
I just want output property to be adjacent.

Reply:
Sometimes it is not recommend, because it increases difficulty to review.  ;)